packages/beacon-node/src/execution/engine/types.ts

```diff
@@ -116,7 +116,7 @@ type ExecutionPayloadRpcWithValue = {
   // even though CL tracks this as executionPayloadValue, EL returns this as blockValue
   blockValue: QUANTITY;
   blobsBundle?: BlobsBundleRpc;
-  requests?: ExecutionRequestsRpc;
+  executionRequests?: ExecutionRequestsRpc;
   shouldOverrideBuilder?: boolean;
```

Comment:
we should probably also reorder these fields to match [the spec](https://github.com/ethereum/execution-apis/blob/main/src/engine/prague.md#engine_getpayloadv4)